index.html

```diff
@@ -14,6 +14,16 @@ <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
+      <nav id="mobile-nav">
+        <button type="button" class="menu-bars">
+          <i class="fas fa-bars"></i>
+        </button>
+        <ul>
+          <li><a class="nav-link mobile-links" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link mobile-links" href="#about-me">About</a></li>
+          <li><a class="nav-link mobile-links" href="#contact">Contact</a></li>
```

Comment:
- [ ] You did a good work creating these links to your page parts, however, you still need to implement their dynamic functionality as asked in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) 🔽 
> When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.